docs/backend/guides/how-to/set-up-a-feature-flag.md

```diff
@@ -9,13 +9,18 @@ There is a feature flag feature for both frontend and backend.  Here's how to se
 
 ### Overview
 
-The frontend feature flag can be turned on or off through the url.  By appending a querystring `?flag:flagName=false`, the feature will be turned off and appending `?flag:flagName=true` will turn on the feature.  Generally, while working on a feature that is not ready for production, by default we set it to off in production and turn it on in dev, staging, and experimental environments.
+The frontend feature flag can be turned on or off through the URL. By appending
+a query string `?flag:flagName=false`, the feature will be turned off and
```

Comment:
@YanZ777 

The issue with Vale was that `url` was not capitalized, and that `querystring` was not two words. I made it multi-line for readability. Sorry for that.